src/game/gamemain.ts

```diff
@@ -0,0 +1,344 @@
+import { Dispatch, SetStateAction } from 'react';
+import Resources from './resources';
```

Comment:
❗️
Отбиваем группы импортов пустой строкой
```suggestion

import Resources from './resources';
```